tests/__mocks__/results/output-reward-split.html

```diff
@@ -1,4 +1,4 @@
 <!-- Ubiquity - GithubCommentModule - GithubCommentModule._callee$ - 1234
```

Comment:
`GithubCommentModule._callee$ - 1234` this is pretty interesting and unexpected but I'm assuming its a mock artifact.

Reply:
Yes `Jest` cannot invoke the `exec` that is used to run the git command to retrieve the commit, so it is mocked returning the hash `1234`. I wanted to make obvious it was mocked by not returning something that could be confused as a real hash.